recipes/compliance-checker/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "2.1.0" %}
+
+package:
+    name: compliance-checker
+    version: {{ version }}
+
+source:
+    fn: compliance-checker-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/c/compliance-checker/compliance-checker-{{ version }}.tar.gz
+    md5: e183eda66050c30f107efbc674538ae0
+
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+    entry_points:
+        - compliance-checker = cchecker:main
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+        - owslib
+        - wicken
+        - lxml
+        - cf_units
+        - requests
+        - python-dateutil
+        - jinja2
+
+test:
+    imports:
+        - compliance_checker
+        - compliance_checker.cf
+        - compliance_checker.tests
+    commands:
+        - compliance-checker --help
+
+about:
+    home: https://github.com/ioos/compliance-checker
+    license:  Apache Software License
```

Comment:
Yep, it's [Apache Software License 2.0]( https://github.com/ioos/compliance-checker/blob/master/LICENSE ).